### ISSUE TITLE
Is this legal 

### ISSUE THREAD

**Issue by AlexFreedomMod** [AlexFreedomRemake/License#1]
We aren't really making our own stuff, we are just making edits to it, Are you sure we can copyright this stuff?

**Comment by AlphaGaming** [comment]
We need to make our own mod to copyright it

**Comment by AlexFreedomMod** [comment]
So we will have to remove this project right now or make our own mod

**Comment by AlphaGaming** [comment]
Yep we have to remove it or make our own mod making our own will be easier to use doe and we have more perms

**Comment by AlphaGaming** [comment]
I can make the self made mod with Netbeans when I get my Pc on Christmas

**Comment by AlphaGaming** [comment]
I need to be owner on this project
